luxonis_train/nodes/backbones/ddrnet/blocks.py

```diff
@@ -0,0 +1,350 @@
+"""DDRNet blocks.
+
+Adapted from: U{https://github.com/Deci-AI/super-gradients/blob/master/src/super_gradients/training/models/segmentation_models/ddrnet.py}
+Original source: U{https://github.com/ydhongHIT/DDRNet}
+Paper: U{https://arxiv.org/pdf/2101.06085.pdf}
+@license: U{https://github.com/Deci-AI/super-gradients/blob/master/LICENSE.md}
+"""
+from typing import Type
+
+import torch
+from torch import Tensor, nn
+
+from luxonis_train.nodes.blocks import ConvModule, UpscaleOnline
+
+
+class DAPPMBranch(nn.Module):
+    def __init__(
+        self,
+        kernel_size: int,
+        stride: int,
+        in_planes: int,
```

Comment:
More general note, but we should rename `planes` to channels everywhere so it's consistent with the rest of the codebase.